### DISCUSSION TITLE
Visual Mode and moving blocks of code up or down

### DISCUSSION THREAD

**Discussion by RizaHKhan** [LunarVim/LunarVim#3280] [General]
Latest release has removed some key-binds?

Ie, can't move block of code up or down in visual mode anymore.

Can anyone give me (and probably others) the snippets we can add to our config so:

1. Highlight line(s) of code
2. Shift + J/K moves the code up/down
Thank you!

**Comment by Leolainen** [comment]
```
lvim.keys.visual_block_mode['J'] = ":move '>+1<CR>gv-gv"
lvim.keys.visual_block_mode['K'] = ":move '<-2<CR>gv-gv"
```
